samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -230,3 +230,27 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp
+  sample.mcumgr.smp_svr.bt.nrf54h20dk:
```

Comment:
Shouldn't we add this in `sdk-nrf/samples/zephyr/subsys/mgmt/mcumgr/smp_svr` ?

Reply:
sdk-nrf, the nrf52840 one here needs to be moved out there also